src/init.cpp

```diff
@@ -906,6 +906,10 @@ bool AppInitParameterInteraction(const ArgsManager& args)
         InitWarning(_("Option '-checkpoints' is set but checkpoints were removed. This option has no effect."));
     }
 
+    if (args.IsArgSet("-datacarriersize") || args.IsArgSet("-datacarrier")) {
```

Comment:
From what I see from previous PR is that after deprecation; the feature will be removed in next release cycle, so maybe indicate that?
https://github.com/bitcoin/bitcoin/pull/31278#discussion_r1838749101

If we will eventually deprecate this then let's indicate that.

Reply:
Personally, I'm fine with the option being subset over a longer period. I'll let other people weigh in on that as a possibility.

Reply:
I think "deprecated" is clear enough, no need to commit to a timeline.